common/djangoapps/enrollment/tests/test_views.py

```diff
@@ -890,10 +894,12 @@ class EnrollmentEmbargoTest(EnrollmentTestMixin, UrlResetMixin, ModuleStoreTestC
     EMAIL = "bob@example.com"
     PASSWORD = "edx"
 
+    URLCONF_MODULES = ['embargo']
```

Comment:
What's the significance of overriding `URLCONF_MODULES` for these embargo tests?